db/migrate/20191005113051_enable_postgis_extensions.rb

```diff
@@ -0,0 +1,21 @@
+class EnablePostgisExtensions < ActiveRecord::Migration[6.0]
+  disable_ddl_transaction!
+
+  def up
+    execute <<-SQL.squish
+      CREATE EXTENSION IF NOT EXISTS postgis WITH SCHEMA public;
+      CREATE SCHEMA IF NOT EXISTS topology;
+      CREATE EXTENSION IF NOT EXISTS postgis_topology WITH SCHEMA topology;
+      CREATE EXTENSION IF NOT EXISTS postgis_sfcgal WITH SCHEMA public;
```

Comment:
this extensions useless for app
postgis extension is created by `rake db:gis:setup`
whole migration must be deleted